README.md

```diff
@@ -1 +1,8 @@
-# developer-checklist
\ No newline at end of file
+# Developer Checklist
+
+## Open the Perfect PR
+1. Create a new branch `your_short_name/feature-description` (Example: `demi/fixing-duplicate-port-bug`). When making a branch, we use our names to distinguish who does the feature.
+1. The PR itself should also begin with `your_short_name/<PR title>` for similar reasons
+    - PR Title should be as short and concise as possible
+1. PRs submitted during your time at Colonist follow a strict format and criteria, both for the [PR description](/docs/pr-format.md), and code quality
+1. Before submitting a PR, ensure it abides by [Developer fundamentals](/docs/developer/developer-fundamentals.md), [Developer Intermediate](/docs/developer/developer-intermediate.md), and [Developer advanced](/docs/developer/developer-advanced.md)
```

Comment:
1. We need to give them instructions on what steps to follow.
2. We don't want the instructions on readme since we want this repo to be a more generic repo for other teams & companies to reffer.

Next steps:
1. Edit this read me explaining what the repo is
2. Create a new doc named `Colonist Take Home Test Opening PR`

Reply:
`Colonist Take Home Test Opening PR` should be more clear.

1. Create an empty repository
2. Create a branch
3. Add the code that you wrote in JSFiddle to that branch
4. Push your code to the repo
5. Create a PR that merges to master
6. Make sure repo is public
7. Make sure everything follows these guides
8. Send the PR link to your contact for review